language/stdlib/modules/designated_dealer.move

```diff
@@ -0,0 +1,150 @@
+address 0x0 {
+    module DesignatedDealer {
```

Comment:
Nit: we usually don't indent inside `address` blocks just to keep the overall indentation reasonable (and because multiple blocks in the same file are uncommon)